desktop-app/app/actions/browser.js

```diff
@@ -284,7 +292,7 @@ export function changeDrawerOpenState(newOpenState) {
   };
 }
 
-export function setPreviewLayout(newLayout) {
+export function setPreviewLayout(newLayout, focusedDeviceId) {
```

Comment:
Can you remove the `focusedDeviceId` arg ?

Reply:
removed